pack/chore.go

```diff
@@ -88,9 +89,20 @@ func (chore *Chore) Run(ctx context.Context) {
 		chore.loop = sync2.NewCycle(chore.interval)
```

Comment:
Also `Run` implies that the function returns only when it has completely finished working. In this case, it's starting in a goroutine, so it should be called `Start` or you should avoid the goroutine in the first place.

Reply:
This is addressed in https://github.com/storj/ipfs-go-ds-storj/pull/68.